src/doc-editors/BackendServicesEditor.vue

```diff
@@ -316,6 +316,27 @@
       </div>
       <span class="is-family-monospace has-text-grey-lighter is-inline-block mt-3">{{ documentAPIPath }}</span>
     </div>
+    <div class="content no-data-wrapper" v-if="loadingDocCounter || !selectedBranch || !selectedBackendService">
+      <div v-if="loadingDocCounter > 0">
+        <button class="button is-outlined is-text is-small is-loading document-loading">
+          Loading
+        </button>
+      </div>
+      <div v-else
+           class="no-data-message">
+        No data found.
+        <div>
+          <!--display correct message by priority (Document type -> Document)-->
+          <span v-if="!docs.find((doc) => doc.id.includes(selectedBackendService?.id))">
```

Comment:
We can simplify that check by checking `selectedBackendService` directly
```suggestion
          <span v-if="!selectedBackendService?.id">
```